tests/wikidata/test_extract.py

```diff
@@ -18,12 +18,20 @@
 @pytest.mark.integration
 def test_search_organization_by_label() -> None:
     """Test organization search in wikidata."""
-    expected = "Q26678"
+    expected = "Q679041"
 
-    search_results = list(search_organization_by_label(item_label="BMW"))
+    search_results = search_organization_by_label(item_label='Robert Koch Institute"')
```

Comment:
better example 👍🙂 

but the variable should now be named `search_result`, its just one org (or None)

Reply:
updated in 108989566acf52f31953594e8d8cbd61be22be7a